lib/scrabble.rb

```diff
@@ -1,18 +1,23 @@
 class Scrabble
+  def initialize
+    @game_reader = GameReader.new
+  end
 
   def score(word)
     1
   end
-
-  def point_values
-    {
-      "A"=>1, "B"=>3, "C"=>3, "D"=>2,
-      "E"=>1, "F"=>4, "G"=>2, "H"=>4,
-      "I"=>1, "J"=>8, "K"=>5, "L"=>1,
-      "M"=>3, "N"=>1, "O"=>1, "P"=>3,
-      "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
-      "U"=>1, "V"=>4, "W"=>4, "X"=>8,
-      "Y"=>4, "Z"=>10
-    }
+  
+  module Points
```

Comment:
Similarly, I don't see the advantage at this point of moving this method to a module. In a professional setting if you walk into an interview given a problem like this and one of the recommendations you make is that you move this into the module, I would want you to be **very** articulate about why that is an important step at this point. It seems tangential to the task at hand.